blt/blt.yml

```diff
@@ -26,12 +26,6 @@ drush:
 tests:
   phpunit:
     - config: '${repo.root}'
-  drupal:
-    web-driver: false
-    sudo-run-tests: false
-    mink-driver-args-webdriver: ''
-    simpletest-db: ''
-    simpletest-base-url: ''
```

Comment:
By removing this, we will use the default settings in `vendor/acquia/blt/config/build.yml` and then we have to override for our local environment.